app/src/main/java/org/oppia/android/app/fragment/FragmentComponent.kt

```diff
@@ -80,6 +81,7 @@ interface FragmentComponent {
   fun inject(autoAppDeprecationNoticeDialogFragment: AutomaticAppDeprecationNoticeDialogFragment)
   fun inject(completedStoryListFragment: CompletedStoryListFragment)
   fun inject(conceptCardFragment: ConceptCardFragment)
+  fun inject(developerOptionsFragment: DeveloperOptionsFragment)
   fun inject(audioLanguageFragment: AudioLanguageFragment)
```

Comment:
Shift this to top.

Reply:
Okay. Should I also shift `DeveloperOptionsActivity` to top in `ActivityComponent`

Reply:
Done for `FragmentComponent`. Waiting for your response about `ActivityComponent`

Reply:
By top I meant just one line above. Basically keep this list in Alphabetical order as much as possible. Same applies for `ActivityComponent` too.

Reply:
There are several instances where this list is not in alphabetical order -- both in `ActivityComponent` and `FragmentComponent`. I suggest we create a separate good first issue for arranging them in alphabetical order. WDYT?

Reply:
Sounds good. Can you file that issue please?

Reply:
Done. PTAL #3378 and #3379